onnx_caffe2/backend.py

```diff
@@ -648,31 +659,33 @@ def onnx_graph_to_caffe2_net(cls, graph_def):
         for node in graph_def.node:
             predict_net.op.extend(cls._onnx_node_to_caffe2_op(node))
 
-        predict_net.external_input.extend(
-            value_info.name for value_info in graph_def.input)
+        if include_inputs_and_initializers:
```

Comment:
This part should not be skipped, otherwise the predict_net would be invalid (inner nodes refer to nonexistent input blobs).